bootstrap-demo/kubernetes-api/src/main/resources/application.conf

```diff
@@ -1,3 +1,6 @@
+akka.loglevel = "INFO"
```

Comment:
To set a good example, maybe use loglevel DEBUG and filter in slf4j/logback? OK though.